client/app/init.js

```diff
@@ -1,8 +1,22 @@
 import { firstbuttonHandler } from './main-page.js';
 import { saveUserInfo } from './save-user.js';
-
+import { renderAllFiles} from './getAll.js'
 document.getElementById('firstpage').addEventListener('click', firstbuttonHandler);
 
 //Post METHOD
-
 document.getElementById('submit').addEventListener('click', saveUserInfo);
+
+// getall METHOD
+
+fetch('/api')
+  .then(res => {
+    if (!res.ok) {
+      throw res;
+    }
+    return res.json();
+  })
+  .then(files => {
+    renderAllFiles(files);
+    
+  })
+  .catch(err => console.error(err));
```

Comment:
@gelilaa Nice job,  Could you place the `fetch get method in getAll.js` then when you click` files button` should execute it otherwise when we add new file we cant see it we need to refresh page to see it.

Reply:
@mametur, I will see what I can do, but I cannot attach it to an event button b/c it will duplicate the render every-time you clicked the show button.

Reply:
Added